tests/CommunityToolkit.Aspire.Hosting.EventStore.Tests/EventStoreFunctionalTests.cs

```diff
@@ -191,7 +204,7 @@ public async Task VerifyWaitForEventStoreBlocksDependentResources()
         var resource = builder.AddEventStore("resource")
                               .WithHealthCheck("blocking_check");
 
-        var dependentResource = builder.AddEventStore("dependentresource")
+        var dependentResource = builder.AddContainer("nginx", "mcr.microsoft.com/cbl-mariner/base/nginx", "1.22")
```

Comment:
Is this a light (meaning small image size and fast startup) container image?
If it is we could use this image for dependent resources in health check tests like this test to speed up our tests.

Reply:
Exactly, I used this specifically because it's lightweight and fast to start.
I've seen usages of this image for the same purpose in the main Aspire repo.